teachers_digital_platform/jinja2/teachers_digital_platform/prototypes/crt-survey.html

```diff
@@ -1,148 +1,339 @@
+                Est. time 30 min.
+            </a>
+        </p>
     </div>
-    <div class="block">
-        <h3>Criterion 1: Name of this criterion</h3>
-        <p>Does the curriculum address grade-level appropriate topics for saving and investing? Select yes if the following components are addressed. Select no if the following components are not addressed.</p>
-        <ol>
-            <li>
-                <h4>Indicator</h4>
+    <div class="block
+                block__flush-top
+                block__flush-bottom">
+        <h2 class="h1">{# {% include icons/document-round.svg %} #}Content</h2>
+        <div class="lead-paragraph
+                    u-mb30">
+            <p>This dimension assesses whether the curriculum content helps students develop knowledge, skills, and behaviors that are important for financial capability. Evaluation criteria are based on research and major national and state education standards.</p>
+        </div>
+        <h3 class="h2">Instructions</h3>
+        <ul>
+            <li>Read through the scope and sequence of the curriculum.</li>
+            <li>Skim the lesson plans.</li>
+            <li>Select <strong><em>yes</em></strong> for the components that are addressed, and <strong><em>no</em></strong> for those that are not.</li>
+        </ul>
+        <div class="m-notification
+                    m-notification__visible
+                    u-mb30
+                    u-mt30
+                    u-mw670">
+            <div class="m-notification_content">
+                <div class="m-notification_message">
+                    <div class="m-notification_message">
+                        <h4>This dimension has <em>essential</em> and <em>beneficial</em> components.</h4>
+                        <p><strong><em>Essential components</em></strong> have been shown to positively impact student learning.<br><strong><em>Beneficial components</em></strong> hold promise for positive impact on student learning, but may be more relevant and useful for some reviewers. Beneficial components are marked with a note; all others are essential. You must answer all components.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <hr class="a-hr
+                    a-hr__gray40
+                    u-mb30
+                    u-mt30">
+    </div>
+    <div class="block block__flush-top">
+        <h3 class="h2">{# {% include icons/approved-round.svg %} #}Criterion 1: Completed criterion <span class="u-fc-gray">(complete)</span></h3>
+        <div class="lead-paragraph
+                    u-mb45">
+            <p>Does the curriculum address grade-level appropriate topics for saving and investing?</p>
+        </div>
+        <div class="o-survey">
+            <div class="o-survey_number">
+                <div class="h3">1.1</div>
+            </div>
+            <div class="o-survey_indicator">
+                <div class="h3">Indicator</div>
```

Comment:
Should this (and "Component") be `<h5 class="h3">`?

Reply:
Fixed

Reply:
@Scotchester Actually, this change will still show some h5-specific styles: letter-spacing, text-shadow, and text-transform. Same thing would happen to any heading classes added to an h6.

I think fixing this would involve updating the mixins in cf-base to use the following in h1-4:
`letter-spacing: initial;
text-shadow: initial;
text-transform: initial`

How do I apply this fix to my project?

Reply:
@cfarm helped me find the solution: I redeclared the mixins and selectors that use those mixins